src/axom/core/tests/utils_Timer.hpp

```diff
@@ -43,53 +49,49 @@ TEST(utils_Timer, timer_check)
 
 TEST(utils_Timer, timer_check_duration)
 {
+  const std::uint32_t sleep_duration_ms = 100;
+  const double sleep_duration_s = sleep_duration_ms / 1000.;
+
   axom::utilities::Timer t;
   t.start();
 
-  sleep(1);
+  sleep_ms(sleep_duration_ms);
 
   t.stop();
-  double e = t.elapsed();
+  const double e = t.elapsed();
   std::cout << "Elapsed: " << e << std::endl;
 
-  EXPECT_GE(e, 1.0);
-  EXPECT_LT(e, 2.0);
+  EXPECT_GE(e, sleep_duration_s);
+  EXPECT_LT(e, 2 * sleep_duration_s);
 }
 
 TEST(utils_Timer, timer_check_sum)
 {
   const int N = 3;
+  const std::uint32_t sleep_duration_ms = 100;
+  const double sleep_duration_s = sleep_duration_ms / 1000.;
+
   axom::utilities::Timer t1(false);
   axom::utilities::Timer t2(false);
   for(int n = 0; n < N; ++n)
   {
     t2.start();
     t1.start();
-    sleep(1);
+    sleep_ms(sleep_duration_ms);
     t1.stop();
-    sleep(1);
+    sleep_ms(sleep_duration_ms);
     t2.stop();
   }
 
-  std::cout << "t1 measured: " << t1.elapsed() << "s in " << t1.cycleCount() << " cycles"
-            << std::endl;
-  std::cout << "t2 measured: " << t2.elapsed() << "s in " << t2.cycleCount() << " cycles"
-            << std::endl;
+  std::cout << "t1 measured: " << t1.elapsed() << "s in " << t1.cycleCount() << " cycles\n";
+  std::cout << "t2 measured: " << t2.elapsed() << "s in " << t2.cycleCount() << " cycles\n";
 
   EXPECT_EQ(t1.cycleCount(), N);
   EXPECT_EQ(t2.cycleCount(), N);
 
-  // Inaccuracy per start-stop cycle, estimated from sample runs.
```

Comment:
@gunney1 -- as we discussed offline, I removed the ad-hoc system-dependent tolerances.

I also reworked this file to use `100 ms` sleep intervals instead of `1 s` intervals since the test was taking too long (8-ish seconds)